client/src/common/types/Course.ts

```diff
@@ -3,6 +3,11 @@
  */
 interface Course {
     name: string;
+    courseID: string;
+    description: string;
+    credithours: number;
+    prereq:  string[];
+    satisfied_reqs: string[];
```

Comment:
What is this for?

Reply:
Those are the university requirements you will be satisfying by taking that course, i.e. Capstone series, or breadth requirements